src/components/Button/Button.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { ButtonHTMLAttributes } from "react";
+
+import { twMerge } from "tailwind-merge";
+
+import { buttonStyles } from "./utils";
```

Comment:
it doesn't make sense the styles coming from "utils"
if we want another file for styles it should be called `styles`.

But I'm okay with declaring the styles here and import them on `ButtonLink`